apps/slackbot/utils.py

```diff
@@ -16,4 +16,49 @@ def load_files(files: List[str]) -> List[Document]:
         documents.extend(loader.load())
     
     print(documents)
-    return documents
\ No newline at end of file
+    return documents
+
+def contains_verbose(query: str) -> bool:
+    '''looks for -verbose in the question and returns True or False \
+    includes -verbosex'''
+    return "-verbose" in query.lower()
+
+def contains_verbosex(query: str) -> bool:
```

Comment:
why are we checking for this?

Reply:
the idea is to have -verbosex give a different response than -verbose